package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mora",
-  "version": "0.7.1",
+  "version": "0.7.1.1",
```

Comment:
Why not just 0.7.2?

Reply:
It's a hotfix that doesn't change anything?

Reply:
I mean, our versioning scheme so far as been major/minor/patch - Last hotfix bumped the version to 0.7.1 so this should be 0.7.2 in my opinion

Reply:
Fair enough — fixed!